Packs/Zoom/Integrations/Zoom/Zoom.py

```diff
@@ -168,5 +641,155 @@ def get_jwt(apiKey, apiSecret):
                 demisto.results('Failed to delete file ' + filename + '.')
     else:
         return_error('Download of recording failed: [%d] - %s' % (res.status_code, res.text))
-else:
-    return_error('Unrecognized command: ' + demisto.command())
+
+
+def zoom_meeting_get_command(client: Client, meeting_id: str, occurrence_id: str = None,
+                             show_previous_occurrences: bool = True) -> CommandResults:
+    # converting
+    show_previous_occurrences = argToBoolean(show_previous_occurrences)
+    # call the API
+    raw_data = client.zoom_meeting_get(meeting_id, occurrence_id, show_previous_occurrences)
+    # parsing the response
+    md = tableToMarkdown('Meeting details', [raw_data], ['uuid', 'id', 'host_id', 'host_email', 'topic',
+                                                         'type', 'status', 'start_time', 'duration',
+                                                         'timezone', 'agenda', 'created_at', 'start_url', 'join_url',
+                                                         ])
+    return CommandResults(
+        outputs_prefix='Zoom.Meetings',
+        readable_output=md,
+        outputs_key_field=str(raw_data["id"]),
```

Comment:
```suggestion
        outputs_key_field='id',
```
this should work, why getting a specific value as key?